specs.json

```diff
@@ -662,6 +662,7 @@
     "properties": {
       "authToken": "string",
       "uri": "string",
+      "sampleRateHz": "number",
```

Comment:
please rename to sampleRate to be consistent with other references in this file

Reply:
fixed